include/vicon_calibration/camera_models/CameraModel.h

```diff
@@ -170,6 +170,17 @@ class CameraModel {
    */
   void WriteJSON(const std::string& file_path,
                  const std::string& method = std::string());
+  
+  /**
+   * @brief Method for retrieving the field of view values of the model
+   * @return fov of the camera
+   */
+  float GetFOV() const;
+
+  /**
+   * @brief Method for calculate field of view
+   */
+  void CalculateFOV();
```

Comment:
add this to the create function so you don't need to remember to call it

Reply:
I wanted, but the create function is static, not allowing variable modification. Then,modifying the create function would lead to more problems...So I just put it in a separate function like this

Reply:
I see, makes sense. Leave as is